ui/src/main/webapp/app/query/input-manager/seed-builder/seed-builder-component.js

```diff
@@ -48,14 +48,19 @@ function seedBuilder() {
 function SeedBuilderController(schema, csv, types, error, events, common, $routeParams, $location) {
     var vm = this;
     vm.seedVertices = '';
+    vm.forceSeedType = false
+    vm.seedType = ''
+    vm.seedTypes = ('Entity Object').split(' ').map(function (seedType) {
```

Comment:
Is there a reason this is so funky?

Reply:
This is "shorthand" for:

```javascript
vm.seedTypes = [
    { value: 'Entity' },
    { value: 'Object' },
]
```

But written as a function in case I needed to extend/adapt this object creation. It's probably fine to replace it with the straightforward array declaration instead if that's preferred?

Reply:
I think the part that looked odd to me was splitting a string with spaces instead of using a list, but honestly both are fine and like you said extensible.